nano/node/nodeconfig.hpp

```diff
@@ -69,14 +71,18 @@ class node_flags
 {
 public:
 	node_flags ();
```

Comment:
This constructor can be `= default` now that you are using in-class member initialization for all members. Although depending on the result below you might not need any constructors other than the implicitly defined default one.

Reply:
`node_flags()` default constructor can be removed